crates/fj-export/src/lib.rs

```diff
@@ -14,16 +14,35 @@
 
 #![deny(missing_docs)]
 
-use std::path::Path;
+use std::{fs::File, path::Path};
+
+use anyhow::{anyhow, Result};
```

Comment:
Since `fj-export` is a library, [`thiserror`](https://crates.io/crates/thiserror) would be a more appropriate choice here for error handling. Doesn't really matter though. Good enough for now!